response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,51 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+
+
+<h1 class="saturn" name="page-messages-title">{{ subject }}</h1>
+
+<div class="secure-message-conversation block">
+    <div class="secure-message-sent-message-meta">
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Survey: </span><span id="survey">{{ messages[0].get('survey') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">RU Ref: </span><span id="ruref">    {{ messages[0].get('ru_ref') }}</span></div>
```

Comment:
Random spacing after ruref id span

Reply:
@insacuri as below, not sure if in HTML we should keep this level of accurancy, and we should idstinguish from suggestion and mandatory changes.